package.json

```diff
@@ -36,7 +36,7 @@
   "dependencies": {
     "@stablelib/chacha20poly1305": "^0.10.2",
     "bignumber.js": "^6.0.0",
-    "bip39": "^2.5.0",
+    "bip39": "2.3.0",
```

Comment:
Why this lib is downgrading?

Reply:
because the newer version adds a checksum test to bip39 mnemonics that does not allow the transformation in the first line of unscrambleStrings, since the entropy we get from this scrambled mnemonic does not have to be valid. Daedalus uses this version of bip39, too, probably for the same reason.

Reply:
Ok, seems that works properly even with the latest version of bip39, so I upgraded it back. I was probably misled by some bug that I had when developing this functionality.